tcms/testruns/tests.py

```diff
@@ -37,7 +36,7 @@ def setUpTestData(cls):
         cls.client = test.Client()
 
     def test_404_if_run_does_not_exist(self):
-        nonexisting_run_pk = TestRun.objects.count() + 1
+        nonexisting_run_pk = 999999
```

Comment:
OK, for some reason ( I think because PKs are global autoincrements on MySQL, generated via LAST_INSERT_ID()) 
```
TestRun.objects.count() == 1
nonexisting_run_pk == 2
```

but the only TestRun object in the database has an actual ID of 2 hence the failure!

On other DB engines the only TestRun object has an ID of 1.  I will update this code with
`TestRun.objects.last().pk + 1` which will be more robust.

However I have found one more thing. In `testruns/urls/run_urls.py` there are two URLs which use the same view:
`.../ordercase/$` and `.../ordercaserun/$`. Do you know why is that? Sounds like a bug (let me know and I will send a separate PR for it).